frontend/src/components/Table/Table.js

```diff
@@ -42,6 +42,9 @@ function TableHeader() {
     },
   ];
 
+  //
```

Comment:
maybe remove this since there are no comments

Reply:
done